### ISSUE TITLE
[Bug]  panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by veo** [projectdiscovery/naabu#191]
**Describe the bug**
can not scan  CIDR 

```
root@n171651:~# git clone https://github.com/projectdiscovery/naabu.git
Cloning into 'naabu'...
remote: Enumerating objects: 1942, done.
remote: Counting objects: 100% (318/318), done.
remote: Compressing objects: 100% (175/175), done.
remote: Total 1942 (delta 163), reused 266 (delta 140), pack-reused 1624
Receiving objects: 100% (1942/1942), 794.26 KiB | 1.08 MiB/s, done.
Resolving deltas: 100% (1120/1120), done.
root@n171651:~# cd naabu/v2/cmd/naabu/
root@n171651:~/naabu/v2/cmd/naabu# go build
root@n171651:~/naabu/v2/cmd/naabu# ./naabu  -host 101.xxx.xxx.97/24 -top-ports top-100

                  __
  ___  ___  ___ _/ /  __ __
 / _ \/ _ \/ _ \/ _ \/ // /
/_//_/\_,_/\_,_/_.__/\_,_/ v2.0.4

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
[INF] Running SYN scan with root privileges
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x79d3c2]

goroutine 1 [running]:
github.com/projectdiscovery/mapcidr.AddressCountIpnet(0x0, 0x11)
        /root/go/pkg/mod/github.com/projectdiscovery/mapcidr@v0.0.6/cidr.go:47 +0x22
github.com/projectdiscovery/ipranger.(*IPRanger).add(0xc0001120a0, 0xc00071c000, 0x11, 0x10, 0x10)
        /root/go/pkg/mod/github.com/projectdiscovery/ipranger@v0.0.3-0.20210619173509-f9b366d18ac6/ipranger.go:77 +0x5a
github.com/projectdiscovery/ipranger.(*IPRanger).Add(0xc0001120a0, 0xc00071c000, 0x11, 0x0, 0x0)
        /root/go/pkg/mod/github.com/projectdiscovery/ipranger@v0.0.3-0.20210619173509-f9b366d18ac6/ipranger.go:65 +0x125
github.com/projectdiscovery/ipranger.(*IPRanger).AddHostWithMetadata(0xc0001120a0, 0xc00071c000, 0x11, 0x9cae31, 0x4, 0xc0000f85a0, 0x0)
        /root/go/pkg/mod/github.com/projectdiscovery/ipranger@v0.0.3-0.20210619173509-f9b366d18ac6/ipranger.go:122 +0xd1
github.com/projectdiscovery/naabu/v2/pkg/runner.(*Runner).AddTarget(0xc0000446c0, 0xc00071c000, 0x11, 0x1, 0x101000000000001)
        /root/naabu/v2/pkg/runner/targets.go:113 +0x59a
github.com/projectdiscovery/naabu/v2/pkg/runner.(*Runner).PreProcessTargets.func1(0xc000716000, 0xc0000446c0, 0xc00071c000, 0x11)
        /root/naabu/v2/pkg/runner/targets.go:97 +0x8e
github.com/projectdiscovery/naabu/v2/pkg/runner.(*Runner).PreProcessTargets(0xc0000446c0, 0x0, 0x0)
        /root/naabu/v2/pkg/runner/targets.go:100 +0x205
github.com/projectdiscovery/naabu/v2/pkg/runner.(*Runner).Load(0xc0000446c0, 0x0, 0x0)
        /root/naabu/v2/pkg/runner/targets.go:29 +0x95
github.com/projectdiscovery/naabu/v2/pkg/runner.(*Runner).RunEnumeration(0xc0000446c0, 0x0, 0x0)
        /root/naabu/v2/pkg/runner/runner.go:114 +0xb4
main.main()
        /root/naabu/v2/cmd/naabu/main.go:18 +0x5d
root@n171651:~/naabu/v2/cmd/naabu# 
```

**Comment by ehsandeep** [comment]
@veo this is a valid bug, thanks for reporting, here is what you can do before we fix this to run the same scan.

```
echo 101.xxx.xxx.97/24 | mapcidr | ./naabu
```

`-top-ports top-100` is the default port to scan for naabu.

We use [mapcidr](https://github.com/projectdiscovery/mapcidr) in naabu to expand the CIDR range.

**Comment by Mzack9999** [comment]
Indirectly fixed by #209
